CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.6.0 2024-04-29
+
+### Changed
+
+- The `/applicationForms` endpoint now returns an `ApplicationFormBundle`.
```

Comment:
Actually, since it's returning a bundle I might as well add the pagination to the endpoint too.

Reply:
Nevermind -- I'd have to write tests.  We can do that some other time.